spec/system/community_sidebar_section_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+describe "Community sidebar section", type: :system do
+  fab!(:user) { Fabricate(:user, locale: "pl_PL") }
+  fab!(:translation_override) do
+    TranslationOverride.create!(
+      locale: "pl_PL",
+      translation_key: "js.sidebar.sections.community.links.topics.content",
+      value: "Tematy",
+    )
```

Comment:
I added translation override as translations for those phrases are not ready. Otherwise, test will start failing when translation arrives